From: Markus Koschany <apo@debian.org>
Date: Mon, 14 Feb 2022 21:55:19 +0100
Subject: disable the h2 console

The h2 console is a developer tool and not required by any reverse-dependency.
Database developers are advised to use the latest version in Debian testing or
unstable. This is a workaround for and mitigation against vulnerabilities like
CVE-2022-23221.
---
 src/main/org/h2/tools/Console.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main/org/h2/tools/Console.java b/src/main/org/h2/tools/Console.java
index da1d05e..d621103 100644
--- a/src/main/org/h2/tools/Console.java
+++ b/src/main/org/h2/tools/Console.java
@@ -96,7 +96,7 @@ ShutdownHandler {
      * @param args the command line arguments
      */
     public static void main(String... args) throws SQLException {
-        new Console().runTool(args);
+        System.out.println("H2 console has been disabled. Please use libh2-java (>= 2.1.210-1) for development purposes.");
     }
 
     /**
